.github/workflows/odh-notebooks-sync.yml

```
@@ -62,10 +62,8 @@ jobs:
               mv $1 Pipfile
               mv $2 Pipfile.lock
             fi
-            if ! pipenv install ${package_name}~="${CODEFLARE_RELEASE_VERSION}"; then
-              echo "Failed to install ${package_name} with version ${CODEFLARE_RELEASE_VERSION} in $dir"
-              exit 1
-            fi
+            # replace existing version of cf-sdk with new version in Pipfile
+            echo "$(sed 's/codeflare-sdk = .*$/codeflare-sdk = \"~=$CODEFLARE_RELEASE_VERSION\"/g' Pipfile | envsubst)" > Pipfile
```

Maybe I'm missing something, but why not to use simply the `sed -i ...` ? E.g.:
```suggestion
            sed --in-place 's/codeflare-sdk = .*$/codeflare-sdk = "~=${CODEFLARE_RELEASE_VERSION}"/g' Pipfile
```
:thinking:

---

Thanks @jstourac, no reason not to. My sed foo is lacking

---

Jk, this doesn't seem to work

---

Right, sorry, I didn't realize :slightly_smiling_face: I put it here without any testing. Thanks for the change!